src/style.css

```diff
@@ -1,3 +1,92 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  height: 60px;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+  position: relative;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.is-active {
+  color: black;
+  text-decoration: none;
+  position: relative;
+  display: inline-block;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 0 0 0 20px;
+  text-align: center;
+}
+
+.nav__link:hover {
+  height: 60px;
+  line-height: 60px;
+  text-decoration: underline;
+  text-decoration-color: var(--main-color);
+  text-decoration-thickness: 2px;
+}
+
+.nav__item:nth-child(4) .nav__link:hover {
+  color: var(--main-color);
+}
```

Comment:
style it by data-qa
<img width="406" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/a1ba3c07-1e90-414a-82c8-832aab5d4a54">